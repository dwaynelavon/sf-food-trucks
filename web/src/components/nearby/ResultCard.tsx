import React from "react";

interface IProps {
    description: string;
    distance: string;
    location: string;
    title: string;
}

const ResultCard = ({ description, distance, location, title }: IProps) => (
    <div className="columns">
        <div className="column is-four-fifths">
            <div className="title is-size-4">{title}</div>
            <div className="subtitle is-size-5">{location}</div>
        </div>
        <div className="column">
            <div className="is-size-5">{distance} mi</div>
        </div>
    </div>
);

export default ResultCard;
