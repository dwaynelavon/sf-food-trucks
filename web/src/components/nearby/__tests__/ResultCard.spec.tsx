import { shallow } from "enzyme";
import React from "react";
import ResultCard from "../ResultCard";

describe("Result Card", () => {
    it("should render with default props", () => {
        const props = {
            description: "description",
            distance: "distance",
            location: "location",
            title: "title",
        };
        const wrapper = shallow(<ResultCard {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});
