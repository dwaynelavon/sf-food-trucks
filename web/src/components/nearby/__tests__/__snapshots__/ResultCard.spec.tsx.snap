// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result Card should render with default props 1`] = `ShallowWrapper {}`;
